ext/node/ops/sqlite/database.rs

```diff
@@ -573,13 +580,204 @@ impl DatabaseSync {
       inner: stmt_cell,
       db: Rc::downgrade(&self.conn),
       statements: Rc::clone(&self.statements),
+      ignore_next_sqlite_error: Rc::clone(&self.ignore_next_sqlite_error),
       use_big_ints: Cell::new(false),
       allow_bare_named_params: Cell::new(true),
       allow_unknown_named_params: Cell::new(false),
       is_iter_finished: Cell::new(false),
     })
   }
 
+  #[fast]
+  #[validate(is_open)]
+  #[undefined]
+  fn function<'a>(
+    &self,
+    scope: &mut v8::PinScope<'a, '_>,
+    #[varargs] args: Option<&v8::FunctionCallbackArguments>,
+  ) -> Result<(), SqliteError> {
+    let Some(args) = args.filter(|args| args.length() > 0) else {
+      return Err(
+        validators::Error::InvalidArgType(
+          "The \"name\" argument must be a string.",
+        )
+        .into(),
+      );
+    };
+
+    if !args.get(0).is_string() {
+      return Err(
+        validators::Error::InvalidArgType(
+          "The \"name\" argument must be a string.",
+        )
+        .into(),
+      );
+    }
+    let name = args.get(0).to_rust_string_lossy(scope);
+
+    let (options_value, function_value) = if args.length() < 3 {
+      (None, args.get(1))
+    } else {
+      (Some(args.get(1)), args.get(2))
+    };
+
+    if !function_value.is_function() {
+      return Err(
+        validators::Error::InvalidArgType(
+          "The \"function\" argument must be a function.",
+        )
+        .into(),
+      );
+    }
+
+    let function: v8::Local<v8::Function> = function_value.try_into().unwrap();
```

Comment:
nit: this can be merged into a let-else. `try_into` returns Err if value is not a function.